azure_functions_worker/dispatcher.py

```diff
@@ -115,13 +116,21 @@ def load_bindings(self):
             self._loop.set_task_factory(
                 lambda loop, coro: ContextEnabledTask(coro, loop=loop))
 
+            # Attach gRPC logging to the root logger
             logging_handler = AsyncLoggingHandler()
             root_logger = logging.getLogger()
             root_logger.setLevel(logging.INFO)
             root_logger.addHandler(logging_handler)
+
+            # Since gRPC channel is established, should use it for logging
+            disable_console_logging()
```

Comment:
What happens to console logs from user code? We still need to flow them as console logs. Can you please test it?

Reply:
Oh, I thought the host will log all messages coming from gRPC channel. For sure, I will test it

Reply:
Yes. Host logs RpcLog messages coming via gRpc. Just to be safe test with a function that logs to console

Reply:
@pragnagopa Testing on my local dev environment, console log (e.g. print()) flows through.

**Synchronous Function Logging**
![image](https://user-images.githubusercontent.com/48038149/77098521-bdb31b80-69cf-11ea-84e4-eede3c673534.png)

**Asynchronous Function Logging**
![image](https://user-images.githubusercontent.com/48038149/77101203-2bf9dd00-69d4-11ea-9611-52232a96bec4.png)